JASP-Engine/JASP/R/bainttestbayesianindependentsamples.R

```diff
@@ -238,9 +238,8 @@ BainTTestBayesianIndependentSamples <- function(jaspResults, dataset, options, .
 
 	  descriptivesTable$addColumnInfo(name="v",                    title = "", type="string")
 		descriptivesTable$addColumnInfo(name="group",                title = "Group", type="string")
-	  descriptivesTable$addColumnInfo(name="N",                    title = "N", type="integer")
+	  descriptivesTable$addColumnInfo(name="N",                    title = "n", type="integer")
 	  descriptivesTable$addColumnInfo(name="mean",                 title = "Mean", type="number")
-	  descriptivesTable$addColumnInfo(name="sd",                   title = "Std. Deviation", type="number")
 	  descriptivesTable$addColumnInfo(name="se",                   title = "Std. Error", type="number")
```

Comment:
Come to think of it I don't think "Std. Error" is an APA abbreviation of standard error -- this should be "SE" or "Standard Error"

Reply:
Should we keep it consistent? Or also change here?
![image](https://user-images.githubusercontent.com/25059399/68838877-d0cf7b00-06bf-11ea-9188-497a555b776f.png)

Reply:
Nice so we have SE, Std. Error and Std. Error of Mean.. :|
Maybe for now just be consistent with all the other t-tests we have and use SE?

Reply:
Alright, will do!